miso-web/src/it/java/uk/ac/bbsrc/tgac/miso/webapp/integrationtest/ListTablesIT.java

```diff
@@ -0,0 +1,260 @@
+    DataTable table = page.getSelectedTable();
+    List<String> headings = table.getColumnHeadings();
+    // size = order columns + some number of completion state columns
+    assertTrue(ordersColumns.size() < headings.size());
+    for (String col : ordersColumns) {
+      assertTrue("Check for column: '" + col + "'", headings.contains(col));
+    }
+  }
+
+  @Test
+  public void testListOrdersColumnSort() throws Exception {
+    testTabbedColumnsSort("poolorders");
+  }
+
+  @Test
+  public void testListContainersSetup() throws Exception {
+    testTabbedPageSetup("containers", containersColumns);
+  }
+
+  @Test
+  public void testListContainersColumnSort() throws Exception {
+    testTabbedColumnsSort("containers");
+  }
+
+  @Test
+  public void testListRunsSetup() throws Exception {
+    testTabbedPageSetup("runs", runsColumns);
+  }
+
+  @Test
+  public void testListRunsColumnSort() throws Exception {
+    testTabbedColumnsSort("runs");
+  }
+
+  @Test
+  public void testListBoxesSetup() throws Exception {
+    testTabbedPageSetup("boxes", boxesColumns);
+  }
+
+  @Test
+  public void testListBoxesColumnSort() throws Exception {
+    testTabbedColumnsSort("boxes");
+  }
+
+  // TODO: re-add once sequencers are listified
+  // @Test
+  // public void testListSequencersSetup() throws Exception {
+  // testTabbedPageSetup("sequencers", sequencersColumns);
+  // }
+  //
+  // @Test
+  // public void testListSequencersColumnSort() throws Exception {
+  // testTabbedColumnsSort("sequencers");
+  // }
+
+  @Test
+  public void testListKitsSetup() throws Exception {
+    testTabbedPageSetup("kitdescriptors", kitsColumns);
+  }
+
+  @Test
+  public void testListKitsColumnSort() throws Exception {
+    testTabbedColumnsSort("kitdescriptors");
+  }
+
+  @Test
+  public void testListIndicesSetup() throws Exception {
+    testTabbedPageSetup("indices", indicesColumns);
+  }
+
+  @Test
+  public void testListIndicesColumnSort() throws Exception {
+    testTabbedColumnsSort("indices");
+  }
+
+  @Test
+  public void testListStudiesSetup() throws Exception {
+    testPageSetup("studies", studiesColumns);
+  }
+
+  @Test
+  public void testListStudiesColumnSort() throws Exception {
+    testColumnsSort("studies");
+  }
+
+  @Test
+  public void testListPrintersSetup() throws Exception {
+    testPageSetup("printers", printersColumns);
+  }
+
+  @Test
+  public void testListPrintersColumnSort() throws Exception {
+    testColumnsSort("printers");
+  }
+
+  private void testPageSetup(String listTarget, Set<String> targetColumns) {
+    ListPage page = getList(listTarget);
+    DataTable table = page.getTable();
+    List<String> headings = table.getColumnHeadings();
+    assertEquals(targetColumns.size(), headings.size());
+    for (String col : targetColumns) {
+      assertTrue("Check for column: '" + col + "'", headings.contains(col));
+    }
+  }
+
+  private void testTabbedPageSetup(String listTarget, Set<String> targetColumns) {
```

Comment:
Should check for presence of expected tabs. I'm not sure if it's possible for the tabs to have different columns, but should check columns on each tab if so